app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<section>
+  <div class="d-flex justify-content-between align-items-center mb-3">
+    <h2 class="my-3">Recipes List</h2>
+    <%= link_to 'Add Recipe', new_recipe_path, class: 'btn btn-outline-secondary w-25' %>
+  </div>
+
+  <%@current_user.recipes.each do |recipe|%>
+    <div class="container border border-4 border-danger p-2 border-opacity-10 rounded mb-3">
+      <div class="row">
+        <div class="col-3 text-center">
+        <a href="/recipes/<%= recipe.id %>" class="link-secondary fs-3"><%= recipe.name %></a>
+        <p><%= button_to "Delete", "recipes/#{recipe.id}" , method: :delete, class: 'btn btn-danger' %></p>
+        </div>
+        <div class="col-6">
+        <p><%=recipe.description%></p>
+        </div>
+      </div>
+    </div>
+  <%end%>
```

Comment:
- [ ] When another user checks for another public recipe it does not match the given wireframe ,kindly fix it, it looks kind of broken on the right side
![Screenshot from 2022-05-19 23-28-16](https://user-images.githubusercontent.com/62705947/169407828-ebf57f49-5165-498d-9e64-b39d939419aa.png)